### ISSUE TITLE
Doesn't work with changed gtk-key-theme

### ISSUE THREAD

**Issue by AD3zFQMq** [p-e-w/plotinus#6]
It works on my computer with a non-changed gtk key theme. However, when I switch it to emacs (which is what I use) it fails to work with Crtl+Shift+P and I couldn't find any other way to get it to work.

**Comment by p-e-w** [comment]
I can't reproduce this. I used GNOME Tweak Tool to change the key theme to "Emacs", then rebooted. However, Plotinus continued working as before, with the same shortcut.

Can you provide more details about your setup?

**Comment by AD3zFQMq** [comment]
Hey, sorry that I took so long. I'm running Arch with the latest updates. I've found that removing my settings.ini with my gtk-key-theme in it (gtk 3.0) allows me to use plotinus. However, I've also found a solution that allows me to keep my emacs gtk-key-theme. In (at least for me) /usr/share/themes/Emacs/gtk-3.0 there is a file called gtk-keys.css. Within that file there is a line saying `bind "<shift><ctrl>p" { "move-cursor" (display-lines, -1, 1) };`. Removing that line seems to fix plotinus. 

Another note: It seems that settings the key theme the gnome way (gconftool-2) doesn't seem to set it in GTK3 and none of the keybindings for Emacs actually applied until I added 'gtk-key-theme-name = Emacs' to my settings.ini in ~/.config/gtk-3.0. That might have interfered with the results for you.
